src/constants/corgi/Sausage.js

```diff
@@ -0,0 +1,8 @@
+const SAUSAGE = {
+  COMMON: 1,
+  UNCOMMON: 10,
+  RARE: 25,
+  VERY_RARE: 50,
```

Comment:
Are these the new corgis width? If yes, can we make them a bit larger? So there is a clear difference between rates. Maybe something like:

```
COMMON: 25
UNCOMMON: 50,
RARE: 100,
VERY_RARE: 200,
```

Reply:
Very good suggestion, looks cooler now!